Automattic/wp-calypso
# Update Woo Blaze Jetpack connection copy

## Proposed Changes

* This PR updates the connect auth and sign-up page copy for Woo Blaze

## Testing Instructions



* Create a WooCommerce site, and make sure the Jetpack plugin is deactivated or not installed
* Point `widgets.wp.com` to your WP Sandbox.
* Run `yarn start` to start calypso 
* Create a plugin build from the woo-blaze plugin repo and install it in your test blog OR run locally as described in repo readme.
* cd `apps/blaze-dashboard` and `yarn dev --sync` to sync blaze dashboard to WP sandbox OR run the commands here https://github.com/Automattic/wp-calypso/pull/90070#issuecomment-2088451908
* In WP Admin, Go to `Marketing` -> `Blaze for WooCommerce`
* Click `Connect now` button and you should see the Jetpack connect flow.


For existing connected users you should see the following connect screen:
<img width="1728" alt="sample-connect" src="https://github.com/Automattic/wp-calypso/assets/9039613/a6009ddc-fe9e-4e24-b6e2-7300a54a08d3">

For a new user/user without a previous connection attempt you should see the following connect screens:
<img width="1728" alt="new user flow" src="https://github.com/Automattic/wp-calypso/assets/9039613/cadec68e-5a93-4d27-a5c4-e9e59c2b8c2c">
<img width="50%" alt="email confirmation" src="https://github.com/Automattic/wp-calypso/assets/9039613/42b09735-37cc-4860-9edc-7c3176006ed5"><img width="50%" alt="email content" src="https://github.com/Automattic/wp-calypso/assets/9039613/bc7d86ea-7d4e-4f58-9056-2231773d7b71">


## Pre-merge Checklist
